src/store/channels/saga.ts

```diff
@@ -31,16 +31,17 @@ export function* markAllMessagesAsRead(channelId, userId) {
     return;
   }
 
-  const status = yield call(markAllMessagesAsReadAPI, channelId, userId);
+  const chatClient = yield call(chat.get);
+  try {
+    yield call([chatClient, chatClient.markRoomAsRead], channelId, userId);
 
-  if (status === 200) {
     yield put(
       receive({
         id: channelId,
         unreadCount: 0,
       })
     );
-  }
+  } catch (error) {}
```

Comment:
It's meaningless at this point because we're killing off Sendbird but the sendbird version of the call returns the http status code (not necessarily throwing an error) so this change isn't backwards compatible.